notebooks/ch11/gradient_descent_define.md

```diff
@@ -128,31 +128,31 @@ plot_tangent_on_loss(pts, 16.5, mse)
 ![png](gradient_descent_define_files/gradient_descent_define_10_0.png)
 
 
-When the slope is positive, decreasing $ \hat{\theta_0} $ will decrease the loss.
+When the slope is positive, decreasing $ \theta $ will decrease the loss.
 
-The slope of the tangent line tells us which direction to move $ \hat{\theta_0} $ in order to decrease the loss. If the slope is negative, we want $ \hat{\theta_0} $ to move in the positive direction. If the slope is positive, $\hat{\theta_0} $ should move in the negative direction. Mathematically, we write:
+The slope of the tangent line tells us which direction to move $ \theta $ in order to decrease the loss. If the slope is negative, we want $ \theta $ to move in the positive direction. If the slope is positive, $\theta $ should move in the negative direction. Mathematically, we write:
 
 $$
-\hat{\theta_{t+1}} = \hat{\theta_t} - \frac{\partial}{\partial \theta} L(\hat{\theta_t}, y)
+\theta^{t+1} = \theta^t - \frac{\partial}{\partial \theta} L(\theta^t, y)
 $$
 
-Where $ \hat{\theta_t} $ is the current estimate and $ \hat{\theta_{t+1}} $ is the next estimate.
+Where $ \theta^t $ is the current estimate and $ \theta^{t+1} $ is the next estimate.
 
 For the MSE, we have:
 
 $$
 \begin{aligned}
 L(\hat{\theta_0}, y)
-&= \frac{1}{n} \sum_{i = 1}^{n}(y_i - \hat{\theta_0})^2\\
-\frac{\partial}{\partial \hat{\theta}} L(\hat{\theta_0}, y)
-&= \frac{1}{n} \sum_{i = 1}^{n} -2(y_i - \hat{\theta_0}) \\
-&= -\frac{2}{n} \sum_{i = 1}^{n} (y_i - \hat{\theta_0}) \\
+&= \frac{1}{n} \sum_{i = 1}^{n}(y_i - \theta)^2\\
+\frac{\partial}{\partial \hat{\theta}} L(\theta, y)
+&= \frac{1}{n} \sum_{i = 1}^{n} -2(y_i - \theta) \\
+&= -\frac{2}{n} \sum_{i = 1}^{n} (y_i - \theta) \\
 \end{aligned}
 $$
 
-When $ \hat{\theta_t} = 12 $, we can compute $ -\frac{2}{n} \sum_{i = 1}^{n} (y_i - \hat{\theta_0}) = -5.2 $. Thus, $ \hat{\theta_{t+1}} = 12 - (-5.2) = 17.2 $.
+When $ \theta^t = 12 $, we can compute $ -\frac{2}{n} \sum_{i = 1}^{n} (y_i - \theta) = -5.2 $. Thus, $ \theta^{t+1} = 12 - (-5.2) = 17.2 $.
```

Comment:
Here / earlier, do we want to use the time notation? I can see both sides :confused:

Reply:
I think the time notation makes sense here since we're performing an explicit update using the gradient descent algorithm.

As a sidenote, the `t` should be in parentheses: `\theta^{(t)}` and `\theta^{(t + 1)}`